scripts/update-tester.php

```diff
@@ -208,9 +208,9 @@ public function testUpdate(
     $drushCmd->dir($absoluteDestinationDocroot);
     $drushCmd->printOutput(FALSE);
     $drushCmd
-      ->arg('status')
+      ->arg('core-status')
       ->option('pipe')
-      ->option('show-passwords');
+      ->option('fields', 'db-password,drupal-version,uri,db-driver,db-hostname,db-port,db-username,db-name,db-status,bootstrap,theme,admin-theme,php-bin,php-conf,php-os,drush-script,drush-version,drush-temp,drush-conf,install-profile,root,site,files,private,temp', '=');
```

Comment:
Could we use "drush core-status --fields=*" ?

Reply:
I was actually thinking the other way around. Just use the fields, that are actually needed. But I guess just providing the same fields as drush 8 does is fine for now. But I do not like to use "*" :-)